[2024-09-22T01:18:47.551+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_eval.group_2_3.20csv manual__2024-09-22T01:18:34.399213+00:00 [queued]>
[2024-09-22T01:18:47.570+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_eval.group_2_3.20csv manual__2024-09-22T01:18:34.399213+00:00 [queued]>
[2024-09-22T01:18:47.571+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-09-22T01:18:47.595+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): group_2_3.20csv> on 2024-09-22 01:18:34.399213+00:00
[2024-09-22T01:18:47.609+0000] {standard_task_runner.py:60} INFO - Started process 5346 to run task
[2024-09-22T01:18:47.613+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_eval', 'group_2_3.20csv', 'manual__2024-09-22T01:18:34.399213+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpj4rf5vxy']
[2024-09-22T01:18:47.615+0000] {standard_task_runner.py:88} INFO - Job 114: Subtask group_2_3.20csv
[2024-09-22T01:18:47.633+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-09-22T01:18:47.706+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_eval.group_2_3.20csv manual__2024-09-22T01:18:34.399213+00:00 [running]> on host cc45b92b61e2
[2024-09-22T01:18:47.880+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_eval' AIRFLOW_CTX_TASK_ID='group_2_3.20csv' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T01:18:34.399213+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T01:18:34.399213+00:00'
[2024-09-22T01:18:47.896+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-09-22T01:18:47.913+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_eval, task_id=group_2_3.20csv, execution_date=20240922T011834, start_date=20240922T011847, end_date=20240922T011847
[2024-09-22T01:18:47.989+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-09-22T01:18:48.044+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
